src/ja/comicgrast/src/eu/kanade/tachiyomi/extension/ja/comicgrast/ComicGrast.kt

```diff
@@ -0,0 +1,109 @@
+package eu.kanade.tachiyomi.extension.ja.comicgrast
+
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.HttpSource
+import eu.kanade.tachiyomi.util.asJsoup
+import keiyoushi.utils.parseAs
+import keiyoushi.utils.tryParse
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.Request
+import okhttp3.Response
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class ComicGrast : HttpSource() {
+    override val name = "Comic Grast"
+    override val baseUrl = "https://novema.jp"
+    override val lang = "ja"
+    override val supportsLatest = false
+
+    private val dateFormat = SimpleDateFormat("yyyy/MM/dd", Locale.JAPAN)
```

Comment:
Pattern doesn't contain any locale specific patterns.
```suggestion
    private val dateFormat = SimpleDateFormat("yyyy/MM/dd", Locale.ROOT)
```